contracts/src/token/erc721/extensions/enumerable.rs

```diff
@@ -137,7 +137,6 @@ pub trait IErc721Enumerable {
     ) -> Result<U256, <Self as IErc721Enumerable>::Error>;
 }
 
-#[public]
 impl IErc721Enumerable for Erc721Enumerable {
```

Comment:
```suggestion
#[public]
#[implements(IErc721Enumerable<Error = Error>, IErc165)]
impl Erc721Enumerable {}

#[public]
impl IErc721Enumerable for Erc721Enumerable {
```
Might make sense to use the conventional implementation, as it would bring a bit more type-safety (we'd be sure the extension implements all the necessary traits)